### PR TITLE
Added a recap side, when reaching the 20'th year.

### PR DESCRIPTION
**Pull request by DominicNr2**
sup sup. Jeg har lavet en recap side, og indsat Frejas flotte hjælpe tekxt UwO

### CODE REVIEW

File: src/main/java/worldofzuul/presentation/RecapController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    @FXML
+    private Label emissionScore, moneyEarnedScore, money, highScore;
+
+    @FXML
+    public void initialize(){
+        int emission = Game.instance.getPlayer().calculateTotalEmission();
+        int moneyEarned = (int)Game.instance.getSoldEnergyPrice();
+        int excessMoney = (int)Game.instance.getPlayer().getPlayerEconomy();
+        int highscore = (50000/(emission/1000))+moneyEarned+excessMoney;
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/RecapController.java]: Integer division here.

File: src/main/java/worldofzuul/presentation/SceneController.java [hunk 2]
```diff
@@ -76,11 +76,22 @@ public void handleBtnHelp() throws Exception {
      * @since 1st Iteration -
      */
     public void handleBtnNextYear() throws Exception {
-        //Change window
-        GUI_Main.setRoot("next year");
-        
-        Command c = new Command(CommandWord.NEXT, "year");
-        Game.instance.nextYear(c);
+        //If 20 years have passed the game is over, and the player will get to the recap
+        if(Game.instance.getGameYear() == 20) {
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/SceneController.java]: You should call the next year method in Game first and check its return value. If it is true show the recap page and if it is false show the next year page

File: src/main/java/worldofzuul/presentation/NextYearController.java [hunk 3]
```diff
@@ -22,7 +22,7 @@
     private Label textNextYear1, yearlyEmissionText, totalEmissionText, earnedMoneyText, balanceText, unableText;
 
     @FXML
-    void initialize(){
+    void initialize() throws IOException {
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/NextYearController.java]: Unused throw

**Review by mathiasgredal — CHANGES_REQUESTED** [review]
Looks pretty good, but there are some logic errors to fix.

**Review by Nolle10 — APPROVED** [review]
LGTM

**Review by mathiasgredal — APPROVED** [review]
LGTM

**Pull request merged**